wd_demo.au3

```diff
@@ -2,6 +2,7 @@
 ; standard UDF's
 #include <ButtonConstants.au3>
 #include <ColorConstants.au3>
+#include <Date.au3>
```

Comment:
The current **master** state is broken until this include is added.
I know this is known, but I want to highlight this situation 🧐 .